site/includes/most-requested-theme-topic.hbs

```diff
@@ -0,0 +1,45 @@
+<div class="container">
+	<div class="row">
+	<div class="pddng-r-0 col-md-2">
+		<h2 class="mrgn-tp-md">
+		{{#is page.language "fr"}}
+		En demande
+		{{else}}
+		Most requested
+		{{/is}}
+		</h2>
+	</div>
+	<div class="col-md-10">
+	<ul class="wb-eqht mrgn-tp-md mrgn-bttm-md colcount-md-2 wb-init wb-eqht-inited">
```

Comment:
Remove CSS added by WET in running mode: wb-init and wb-eqht-inited

Reply:
wb-init removed 
wb-eqht-inited removed